### DISCUSSION TITLE
Translated changelog

### DISCUSSION THREAD

**Discussion by TrianguloY** [TrianguloY/OpenInWhatsapp_Translation#30] [Q&A]
This is a message to all translators of the app. 
@solokot an @stevenfelix505, sorry for the mention, but since you usually provide updates regularly I though you are the most interested (if you prefer not to be notified please tell me).

For the new version I plan to add a little changelog dialog, which will be shown when clicking the version number from the about dialog or when opening the app after an update (configurable). This dialog will show the changelog of the latest version as text.

Currently the changelog is only shown in English and Spanish (translations made by me) and I don't know if it should be a good idea to let the changelog be translatable. On one hand having a translatable changelog means the Play Store will also show it translated, but on the other hand this will only be possible if you provide it before the update, preferably in the public beta process (extra note: tell me if you want to be notified on new updates).

Any thoughts? Suggestions?

**Comment by stevenfelix505** [comment]
It will be useful for people who cannot understand English.

**Comment by TrianguloY** [comment]
Well, yes of course. The question was more asking for: do you mind translating the changelog for every new version?

I think I'll leave the option to translate it, but use English if a translation can't be made or isn't found (this is what I've been doing for the rest of the strings since the beginning anyway).

**Comment by stevenfelix505** [comment]
Of course I don't mind translating the changelog for each latest version.

**Comment by solokot** [comment]
To be honest, I don't really want to translate the history of changes. In my opinion, it is read like an instruction manual for the elevator (when stuck in it).

**Comment by TrianguloY** [comment]
Don't worry, I understand. The changelog can be useless for some people, but it also is very useful for others (myself included). That's why I added it, but making sure that it can be disabled.

You are free to translate whatever you want, as usual, so as I said I'll leave the option to translate it but at the same time revert to english if not found. This should please both of you.

**Comment by stevenfelix505** [comment]
Update :
now i change my mind, i will translate the changelog, please check my [pull request](https://github.com/TrianguloY/OpenInWhatsapp_Translation/pull/43)

**Reply by TrianguloY** [reply]
Saw it! Any work is enough, so thanks for your effort translating the rest of the files.
Also, don't feel pressured to translate future changelogs, just do as you want.

Let me a few days to review and I'll merge it. Thanks again!
